src/libraries/System.Runtime/tests/System.IO.FileSystem.Tests/Directory/GetFiles.cs

```diff
@@ -192,5 +192,45 @@ public void DirectoryWithTrailingSeparators(string trailing)
             string[] files = GetEntries(root + (IsDirectoryInfo ? trailing : ""), "*", SearchOption.AllDirectories);
             FSAssert.EqualWhenOrdered(new string[] { rootFile, nestedFile }, files);
         }
```

Comment:
Could you please make sure that the GetFiles tests include the scenario described in https://github.com/dotnet/runtime/issues/113120 ?

Note that this scenario does not work today. Please verify that the test is failing and add it as disabled using `ActiveIssue` annotation.